pegasus/sites.v3/code.org/styles_min/040-page.css

```diff
@@ -121,12 +123,26 @@ h6,
   margin-bottom: 0.5em;
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   text-wrap: balance;
 }
 
 /* Use this class on text elements to turn them white */
-h1, h2, h3, h4, h5, h6, p, figcaption, span, a:not(.link-button) {
+h1,
```

Comment:
Nothing changed here; this got formatted on save.

Reply:
Nit: Do you have 'format on save' applying to the whole file? I have mine set to 'modifications' to avoid having to deal with what happens in situations like this😅

Reply:
I didn't know that was an option 🙀 Setting it now!